src/helpers/utils.js

```diff
@@ -0,0 +1,65 @@
+import { WrongElementTypeException } from 'exceptions/element';
+
+/**
+ * Remove the trailing slash from an URL and return it
+ * @param {string} url
+ * @return {string} - The cleaned URL
+ */
+export function removeTrailingSlash(url) {
+  return url.replace(/\/*$/, '');
```

Comment:
Technically you remove all the trailing slashes (plural). So maybe `removeTrailingSlashes`.

Reply:
Yep, you're right I'll fix that